src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -858,7 +858,7 @@ void AssertFileExists(string fileName)
 
                 string versionRegex = @"8.0.\d?(-[a-z]+(\.\d\.\d+\.\d)?)?\.([a-z0-9])+";
                 Assert.Collection(
-                    dotnetJsEntries,
+                    dotnetJsEntries.OrderBy(f => f),
```

Comment:
This depends on order https://github.com/dotnet/aspnetcore/blob/690d78279e940d267669f825aa6627b0d731f64c/src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts#L202 which is less than ideal.  I guess this code would be gone soon.

Reply:
I see, that will go away in few days